orc8r/cloud/go/services/configurator/storage/sql_entity_graph_helpers.go

```diff
@@ -23,53 +23,54 @@ import (
 )
 
 type internalEntityGraph struct {
-	entsByPk map[string]*NetworkEntity
-	edges    []loadedAssoc
+	entsByTK EntitiesByTK
+	edges    loadedAssocs
 }
 
-// loadGraphInternal will load all entities and assocs for a given graph ID.
-// This function will NOT fill entities with associations.
+// loadGraphInternal will load all loadEntities and assocs for a given graph ID.
+// This function will NOT fill loadEntities with associations.
 func (store *sqlConfiguratorStorage) loadGraphInternal(networkID string, graphID string, criteria EntityLoadCriteria) (internalEntityGraph, error) {
 	loadFilter := EntityLoadFilter{GraphID: &wrappers.StringValue{Value: graphID}}
-	entsByPk, err := store.loadFromEntitiesTable(networkID, loadFilter, criteria)
+	ents, err := store.loadEntities(networkID, loadFilter, criteria)
 	if err != nil {
 		return internalEntityGraph{}, errors.Wrap(err, "failed to load entities for graph")
 	}
-	if funk.IsEmpty(entsByPk) {
+	if funk.IsEmpty(ents) {
```

Comment:
This is from Jacky. I went back and forth on this, but have tentatively landed on "IsEmpty" and "NotEmpty" are more readable. Once we have scale test framework we can benchmark how slow it is. If it's non-trivially slower, we can remove their usages

But I'm expecting it to be not too slow, especially since it's not in a tight loop. So tentatively in favor of the added readability